tests/client/disco/components/TestAddon.js

```diff
@@ -62,16 +65,59 @@ describe('<Addon />', () => {
       root = renderAddon(result);
     });
 
-    it('renders a default error overlay', () => {
+    it('renders a default error overlay with no close link', () => {
       const data = { ...result, status: ERROR, setCurrentStatus: sinon.stub() };
       root = renderAddon(data);
       const error = findDOMNode(root).querySelector('.notification.error');
       assert.equal(
         error.querySelector('p').textContent,
         'An unexpected error occurred.',
         'error message should be present');
-      Simulate.click(error.querySelector('.close'));
-      assert.ok(data.setCurrentStatus.called, 'setCurrentStatus should be called');
+    });
+
+    it('renders a default error overlay with no close link for FATAL_ERROR', () => {
+      const data = {
+        ...result,
+        status: ERROR,
+        setCurrentStatus: sinon.stub(),
+        error: FATAL_ERROR,
+      };
+      root = renderAddon(data);
+      const error = findDOMNode(root).querySelector('.notification.error');
+      assert.equal(
+        error.querySelector('p').textContent,
+        'An unexpected error occurred.',
+        'error message should be present');
+    });
+
+    it('renders a specific overlay with no close link for FATAL_INSTALL_ERROR', () => {
+      const data = {
+        ...result,
+        status: ERROR,
+        setCurrentStatus: sinon.stub(),
+        error: FATAL_INSTALL_ERROR,
+      };
+      root = renderAddon(data);
+      const error = findDOMNode(root).querySelector('.notification.error');
+      assert.equal(
+        error.querySelector('p').textContent,
+        'An unexpected error occurred during installation.',
+        'error message should be present');
+    });
+
+    it('renders a specific overlay with no close link for FATAL_UNINSTALL_ERROR', () => {
+      const data = {
+        ...result,
+        status: ERROR,
+        setCurrentStatus: sinon.stub(),
+        error: FATAL_UNINSTALL_ERROR,
+      };
+      root = renderAddon(data);
+      const error = findDOMNode(root).querySelector('.notification.error');
+      assert.equal(
+        error.querySelector('p').textContent,
+        'An unexpected error occurred during uninstallation.',
+        'error message should be present');
     });
 
```

Comment:
> You don't test throwing an error that is not fatal (which would show the close link), is that done elsewhere ?

That is covered elsewhere.

> Also, you don't actually test that there is no close link in those tests you added, despite mentioning it in the test string ?

Yep, good point, I'm adding some assertions for that.